libraries/chain/database.cpp

```diff
@@ -3809,6 +3809,19 @@ void database::apply_hardfork( uint32_t hardfork )
       case STEEMIT_HARDFORK_0_18:
          break;
       case STEEMIT_HARDFORK_0_19:
+         {
+            const auto& delegation_idx = get_index< vesting_delegation_index, by_id >();
+            auto itr = delegation_idx.begin();
+
+            while( itr != delegation_idx.end() )
```

Comment:
Might be worth optimizing this using Boost's `erase(iterator first,iterator last)` if we order by delegated vesting shares, or just `erase(const key_type& x)` if we keep vesting shares a member in the index.